prebuilt-ui/rtp-tracks-compositing/public/index.html

```diff
@@ -0,0 +1,207 @@
+<html>
+  <head>
+    <title>Instructor View</title>
+    <script src="https://unpkg.com/@daily-co/daily-js"></script>
+  </head>
+
+  <body onload="main()">
+    <div id="buttons">
+      <button id="start" onclick="startSession()">Start Session</button>
+      <button id="stop" onclick="stopSession()">Stop Session</button>
+      <span id="recording-url"></span>
+    </div>
+    <script>
+      var recordingId = '';
+
+      async function main() {
+      	window.call = DailyIframe.createFrame({
+      		url: 'YOUR_ROOM_URL_HERE',
+      		showLeaveButton: true,
+      		iframeStyle: {
+      			position: 'fixed',
+      			bottom: '0',
+      			left: '0',
+      			width: '100%',
+      			height: '90%',
+      		},
+      		dailyConfig: {
+      			experimentalChromeVideoMuteLightOff: true,
+      		},
+      	});
+
+      	call.on('recording-started', (e) => {
+      		console.log(
+      			'Recording started, id: ',
+      			e.recordingId,
+      			', Instructor session: ',
+      			window.instructorSessionId
+      		);
+      		recordingId = e.recordingId;
+      	});
+
+      	call.on('recording-stopped', (e) => {
+      		console.log('Recording stopped');
+      	});
+
+      	call.on('left-meeting', (e) => {
+      		// might want to just start processing the recording after the meeting starts in case the instructor closes the window
+      		// or use webhook-meeting-end to process recording
+      		console.log('Leaving meeting');
+      	});
+
+      	window.token = await createMeetingToken({
+      		enable_recording: 'rtp-tracks',
+      		user_name: 'Instructor',
+      		user_id: 'instructor',
+      		is_owner: true,
+      	});
+
+      	let participants = await call.join({
+      		token: token
+      	});
+
+      	call.setLocalVideo(true);
+      	call.setActiveSpeakerMode(false);
+
+      	window.instructorSessionId = participants.local.session_id;
+      }
+
+      async function createMeetingToken(properties) {
+      	let response = await fetch('/api/daily/meeting-tokens', {
+      		method: 'post',
+      		body: JSON.stringify({
+      			properties
+      		}),
+      	});
+      	return (await response.json()).token;
+      }
+
+      async function startSession() {
+      	call.sendAppMessage({
+      		message: 'startSession'
+      	}, '*');
+      	call.startRecording();
+      }
+
+      async function stopSession() {
+      	call.sendAppMessage({
+      		message: 'stopSession'
+      	}, '*');
+      	call.stopRecording();
+      	console.log('Starting processing for recording id:', recordingId);
+      	processRecording(￼)
```

Comment:
![image](https://user-images.githubusercontent.com/3722211/123647042-bfd07080-d7f5-11eb-9418-95873019a79f.png)
I'm assuming this is an emoji?